nautobot/docs/models/extras/dynamicgroup.md

```diff
@@ -1,7 +1,483 @@
+
+From an individual object's detail page, if it is a member of any groups, a "Dynamic Groups" tab will display in the navigation tabs. Clicking that tab will display all Dynamic Groups of which this object is a member.
+
+### Filtering
+
+Dynamic Group filtering is powered by **FilterSet** objects underneath the hood. Basic filtering is performed using the `filter` that is defined on a given Dynamic Group. Advanced filtering is performed by aggregating filters from multiple nested Dynamic Groups to form a combined parent Dynamic Group, which will be explained later in this document.
+
+An object is considered to be a member of a Dynamic Group if it is of the same Content Type and it is not excluded by way of any of the filter criteria specified for that group. By default, a freshly created group has an empty filter (`{}`), which will include all objects of the matching Content Type, just as a default list view of those objects would display prior to any filter fields being filled in the web UI.
+
+For example, for a Dynamic Group with Content Type of `dcim.device` and an empty filter, the list of members would be equivalent to the default Device list view, which in turn is equivalent to the queryset for `Device.objects.all()` from the database ORM.
+
+!!! warning
+    _Changed in version 1.4.0_  <!-- markdownlint-disable-line MD036 -->
+
+    v1.3.0 the default for a group with an empty filter was to fail "closed" and have zero members.
+
+    As of v1.4.0, this behavior has been inverted to default to include all objects matching the Content Type, instead of matching no objects as was previously the case. This was necessary to implement the progressive layering of child filters similarly to how we use filters to reduce desired objects from basic list view filters. This will described in more detail below.
+
+#### Basic Filtering
+
+When editing a Dynamic Group, under the **Filter Options** section, you will find a **Filter Fields** tab that allows one to specify filter criteria. The filter fields available for a given Content Type are backed and validated by underlying filterset classes (for example `nautobot.dcim.filters.DeviceFilterSet`) and are represented in the web interface as a dynamically-generated filter form that corresponds to each eligible filter field.
+
+(TODO: screenshots here)
+
+#### Advanced Filtering
+
+_Added in version 1.4.0_  <!-- markdownlint-disable-line MD036 -->
+
+Advanced filtering is performed using nested Dynamic Group memberships.
+
+An object is considered a member of an advanced Dynamic Group if it matches the aggregated filter criteria across all descendant groups.
+
+When editing a Dynamic Group, under the **Filter Options** section, you will find a **Child Groups** tab that allows one to make other Dynamic Groups of the same Content Type children of the parent group.
+
+![Child Groups](../../media/models/dynamicgroup02.png)
```

Comment:
Looks like this is the only one of `dynamicgroup0[1-5].png` that we're using now - can we delete the other four and rename this to something like "dynamicgroup_filtering_01.png` instead?